matrix_70X.txt

```diff
@@ -16,17 +16,27 @@ TTbarwf25p0 25.0_TTbar+TTbarINPUT+DIGI+RECOAlCaEle+HARVEST+ALCATT/step3.root REC
+RunMinBias2011Awf4p17 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3.root reRECO 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3_inMINIAOD.root reRECO
 RunMinBias2011Awf4p17 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT+MINIAODDreHLT/step3.root reRECO
-RunMinBias2011Awf4p17 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODR1reHLT+HARVESTDR1reHLT+SKIMDreHLT/step3.root reRECO
+RunMinBias2011Awf4p17 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT+MINIAODDreHLT/step6.root PAT
 RunCosmics2011Awf4p22 4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2.root reRECO
 RunMinBias2011Bwf4p291 4.291_RunMinBias2011B+RunMinBias2011B+HLTD+RECODst3+HARVESTD/step3.root reRECO
-RunMinBias2011Bwf4p29 4.29_RunMinBias2011B+RunMinBias2011B+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3.root reRECO
+RunMinBias2011Bwf4p29 4.29_RunMinBias2011B+RunMinBias2011B+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3.root reRECO 4.29_RunMinBias2011B+RunMinBias2011B+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3_inMINIAOD.root reRECO
 RunMinBias2011Bwf4p29 4.29_RunMinBias2011B+RunMinBias2011B+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT+MINIAODDreHLT/step3.root reRECO
+RunMinBias2011Bwf4p29 4.29_RunMinBias2011B+RunMinBias2011B+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT+MINIAODDreHLT/step6.root PAT
 RunMinBias2011Bwf4p29 4.29_RunMinBias2011B+RunMinBias2011B+HLTD+RECODR1reHLT+HARVESTDR1reHLT+SKIMDreHLT/step3.root reRECO
 ZMuSkim2011Bwf4p37 4.37_ZMuSkim2011B+ZMuSkim2011B+RECOSKIM+HARVESTD/step2.root reRECO
-ZMuSkim2011Bwf4p37 4.37_ZMuSkim2011B+ZMuSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT/step3.root reRECO
+ZMuSkim2011Bwf4p37 4.37_ZMuSkim2011B+ZMuSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT/step3.root reRECO 4.37_ZMuSkim2011B+ZMuSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT/step3_inMINIAOD.root reRECO
 ZMuSkim2011Bwf4p37 4.37_ZMuSkim2011B+ZMuSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step3.root reRECO
+ZMuSkim2011Bwf4p37 4.37_ZMuSkim2011B+ZMuSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step5.root PAT
+ZElSkim2011Bwf4p38 4.38_ZElSkim2011B+ZElSkim2011B+RECOSKIM+HARVESTD/step2.root reRECO
+ZElSkim2011Bwf4p38 4.38_ZElSkim2011B+ZElSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT/step3.root reRECO
+ZElSkim2011Bwf4p38 4.38_ZElSkim2011B+ZElSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step3.root reRECO
+ZElSkim2011Bwf4p38 4.38_ZElSkim2011B+ZElSkim2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step5.root PAT
+HighMet2011Bwf4p39 4.39_HighMet2011B+HighMet2011B+RECOSKIM+HARVESTD/step2.root reRECO
+HighMet2011Bwf4p39 4.39_HighMet2011B+HighMet2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT/step3.root reRECO
+HighMet2011Bwf4p39 4.39_HighMet2011B+HighMet2011B+HLTDSKIM+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step3.root reRECO
```

Comment:
the 3 above already appear in the master.
I don't expect them to show up as additions in the diffs.
I think the reason is that you have commits that re-add these entries.
You should have rebased to the master instead of re-applying the changes.
Please rebase to the master  instead of merging and cleanup redundant commits.